src/plugins/intel_gpu/src/kernel_selector/tensor_type.h

```diff
@@ -625,6 +646,13 @@ struct DataTensor : public TensorBaseT<Datatype, DataLayout> {
     DataTensor FlattenFeatureAndSpatials() const;
     DataTensor FlattenEverything() const;
     void SwapXY();
+    void SetDynamicShapeOffset(size_t offset) {
+        dynamic_shape_offset = offset;
+    }
+
+    size_t GetDynamicShapeOffset() const {
```

Comment:
nit: `GetDynamicShapeOffset` -> `get_dynamic_shape_offfset` (refer to https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/coding_style.md for coding style. Ideally new methods/classes should follow it)

Reply:
fixed